cmake/configure/configure_1_cuda.cmake

```diff
@@ -25,29 +25,60 @@ SET(FEATURE_CUDA_DEPENDS CXX11)
 
 MACRO(FEATURE_CUDA_FIND_EXTERNAL var)
 
-  #
-  # FIXME Restructure this call into a ./modules/FindCUDA.cmake file and
-  # use DEAL_II_PACKAGE_HANDLE
-  #
-  FIND_PACKAGE(CUDA)
-  MARK_AS_ADVANCED(
-    CUDA_HOST_COMPILER
-    CUDA_SDK_ROOT_DIR
-    CUDA_TOOLKIT_ROOT_DIR
-    CUDA_USE_STATIC_CUDA_RUNTIME
-    )
+  IF(${DEAL_II_WITH_CUDA})
+    #
+    # FIXME Restructure this call into a ./modules/FindCUDA.cmake file and
+    # use DEAL_II_PACKAGE_HANDLE
+    #
+    FIND_PACKAGE(CUDA)
+    MARK_AS_ADVANCED(
+      CUDA_HOST_COMPILER
+      CUDA_SDK_ROOT_DIR
+      CUDA_TOOLKIT_ROOT_DIR
+      CUDA_USE_STATIC_CUDA_RUNTIME
+      )
 
-  IF(CUDA_FOUND)
+    IF(CUDA_FOUND)
 
-    SET(CUDA_ATTACH_VS_BUILD_RULE_TO_CUDA_FILE FALSE)
+      IF("${CUDA_NVCC_FLAGS}" MATCHES "-arch")
 
-    SET(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS} -std=c++11)
+        # Compute Capability specified explicitly.
+        # Now parse:
 
-    # FIXME: CUDA compiler NVCC doesn't support C++14.
+        IF("${CUDA_NVCC_FLAGS}" MATCHES "-arch[ ]*sm_([0-9]*)")
+          SET(CUDA_COMPUTE_CAPABILITY "${CMAKE_MATCH_1}")
+        ELSEIF("${CUDA_NVCC_FLAGS}" MATCHES "-arch=sm_([0-9]*)")
+          SET(CUDA_COMPUTE_CAPABILITY "${CMAKE_MATCH_1}")
+        ELSE()
+          MESSAGE( FATAL_ERROR "Missformed Compute Capability specified")
```

Comment:
Is it necessary to fatal error here? Can't we simply set `SET(${var} FALSE)` and not configure cuda?